MorkoUnityProject/Assets/Scripts/LocalController.cs

```diff
@@ -158,38 +108,69 @@ public AvatarPackage Update()
 				character.transform.rotation = Quaternion.RotateTowards(lookRotation, character.transform.rotation, Time.deltaTime);
 			}
 			// Controller being used, right joystick not being used, look towards player forward
-			if (moveDirectionKeyboard != Vector3.zero && joystickRotateX == 0 && joystickRotateY == 0 && mouseDelta.x == 0 && mouseDelta.y == 0 && mouseRotate == false)
+			if (moveDirection != Vector3.zero && joystickRotateX == 0 && joystickRotateY == 0 && mouseDelta.x == 0 && mouseDelta.y == 0 && mouseRotate == false)
 			{
-				character.transform.rotation = Quaternion.LookRotation(moveDirectionKeyboard);
+				character.transform.rotation = Quaternion.LookRotation(moveDirection);
 			}
 			// Mouse being used, keep old rotation
 			if (joystickRotateX == 0 && joystickRotateY == 0 && mouseDelta.x == 0 && mouseDelta.y == 0 && mouseRotate == true)
 			{
 				character.transform.rotation = package.rotation;
 			}
 			
-			CalculateMovementSpeed();
+			// Hardcoded min magnitude value where joystick is maxed out
+			bool joystickMaxed = moveDirection.magnitude >= 0.8;
+			
+			// Sneak
```

Comment:
This comment (and ones below) can be removed:
```
bool sneak = /* Sneak condition*/ ;
bool accelerateWalk = /* accelerate walk condition */;
if (sneak) 
{
    // Sneak code
}
else if (accelerateWalk)
{
    // Code...
}
```
Less comments and more well named variables is better since compiler will check variables on change, but not comments, and some context could be lost when code is changed